### PR TITLE
Add routine for reconstructing original matrix given Cholesky decompo…

### PR DESCRIPTION
**Pull request by btracey**
…sition

### CODE REVIEW

**Comment by kortschak** [comment]
Why is this needed?

**Comment by btracey** [comment]
PR coming shortly to distmv. In the middle of writing tests.

**Comment by btracey** [comment]
Sorry for the delay. I was having trouble getting correct code.

File: mat64/cholesky.go [hunk 1]
```diff
@@ -142,3 +142,11 @@ func (t *TriDense) LFromCholesky(chol *Cholesky) {
 	t.reuseAs(n, blas.Lower)
 	t.Copy(chol.chol.TTri())
 }
+
+// FromCholesky reconstructs the original positive definite matrix given its
+// cholesky decomposition.
```
Comment (vladimir-ch) [inline mat64/cholesky.go]: capital C

**Comment by vladimir-ch** [comment]
LGTM

**Pull request merged**